internal/saucecloud/cloud.go

```diff
@@ -128,25 +131,31 @@ func (r *CloudRunner) runJobs(jobOpts <-chan job.StartOptions, results chan<- re
 	}
 }
 
-func (r CloudRunner) archiveAndUpload(project interface{}, files []string) (string, error) {
+func (r CloudRunner) archiveAndUpload(project interface{}, files []string, configDir string) (string, error) {
```

Comment:
It takes in a `configDir` but you are using it like `r.archiveAndUpload(r.Project, files, r.Project.Sauce.Sauceignore)`.
`r.Project.Sauce.Sauceignore` is not a config dir though ;)

Reply:
I renamed `configDir` -> `sauceignoreFile`